### ISSUE TITLE
No copyright declared in source code

### ISSUE THREAD

**Issue by mr-c** [common-workflow-language/cwltool#35]

**Comment by manu-chroma** [comment]
something like this? https://github.com/common-workflow-language/avro/blob/cwl/src/avro/__init__.py

**Comment by mr-c** [comment]
Yes, but we don't use contributor license agreements nor is CWL a legal entity like the ASF yet

**Comment by manu-chroma** [comment]
Interesting, can you point me to something more along the lines of what we might like to put?

**Comment by mr-c** [comment]
This isn't a high priority right now
